`TestIntegrations/Disconnection/expired_cert_node_recording` flakiness

## Failure

#### Link(s) to logs

- https://github.com/gravitational/teleport/actions/runs/7348909336/job/20009131879?pr=36079

#### Relevant snippet

```
=== FAIL: integration TestIntegrations/Disconnection/expired_cert_node_recording (2.30s)
    integration_test.go:2306: 
        	Error Trace:	/__w/teleport/teleport/integration/integration_test.go:2306
        	            				/__w/teleport/teleport/integration/integration_test.go:2197
        	Error:      	Async error
        	Test:       	TestIntegrations/Disconnection/expired_cert_node_recording
        	Messages:   	expected EOF, ExitError, or nil, got failed connecting to host localhost:35307: failed to receive cluster details response
        	            		failed to dial target host
        	            		direct dialing to nodes not found in inventory is not supported instead
        --- FAIL: TestIntegrations/Disconnection/expired_cert_node_recording (2.30s)
```

Different failure here: https://github.com/gravitational/teleport/actions/runs/7614729939/job/20737699334

```
    integration_test.go:2310: 
        	Error Trace:	/__w/teleport/teleport/integration/integration_test.go:2310
        	            				/__w/teleport/teleport/integration/integration_test.go:2200
        	Error:      	timeout
        	Test:       	TestIntegrations/Disconnection/expired_cert_node_recording
        	Messages:   	Jan 22 17:11:41.975 timeout waiting for session to exit: {name:expired cert node recording recordingMode:node options:{ForwardAgent:false MaxSessionTTL:2000000000 PortForwarding:<nil> CertificateFormat: ClientIdleTimeout:0 DisconnectExpiredCert:true BPF:[] PermitX11Forwarding:false MaxConnections:0 MaxSessions:0 RequestAccess: RequestPrompt: Lock: RecordSession:<nil> DesktopClipboard:<nil> CertExtensions:[] MaxKubernetesConnections:0 DesktopDirectorySharing:<nil> CreateHostUser:<nil> PinSourceIP:false SSHFileCopy:<nil> RequireMFAType:0 DeviceTrustMode: IDP:<nil> CreateDesktopUser:<nil> CreateDatabaseUser:<nil> CreateHostUserMode:0 CreateDatabaseUserMode:0 XXX_NoUnkeyedLiteral:{} XXX_unrecognized:[] XXX_sizecache:0} disconnectTimeout:4000000000 concurrentConns:1 sessCtlTimeout:0 postFunc:<nil> clientConfigOpts:<nil> verifyError:<nil>}
        --- FAIL: TestIntegrations/Disconnection/expired_cert_node_recording (7.25s)
```

https://github.com/gravitational/teleport/actions/runs/7619197716/job/20751846578

```
    integration_test.go:2310: 
        	Error Trace:	/__w/teleport/teleport/integration/integration_test.go:2310
        	            				/__w/teleport/teleport/integration/integration_test.go:2200
        	Error:      	timeout
        	Test:       	TestIntegrations/Disconnection/expired_cert_node_recording
        	Messages:   	Jan 23 00:33:15.930 timeout waiting for session to exit: {name:expired cert node recording recordingMode:node options:{ForwardAgent:false MaxSessionTTL:2000000000 PortForwarding:<nil> CertificateFormat: ClientIdleTimeout:0 DisconnectExpiredCert:true BPF:[] PermitX11Forwarding:false MaxConnections:0 MaxSessions:0 RequestAccess: RequestPrompt: Lock: RecordSession:<nil> DesktopClipboard:<nil> CertExtensions:[] MaxKubernetesConnections:0 DesktopDirectorySharing:<nil> CreateHostUser:<nil> PinSourceIP:false SSHFileCopy:<nil> RequireMFAType:0 DeviceTrustMode: IDP:<nil> CreateDesktopUser:<nil> CreateDatabaseUser:<nil> CreateHostUserMode:0 CreateDatabaseUserMode:0 XXX_NoUnkeyedLiteral:{} XXX_unrecognized:[] XXX_sizecache:0} disconnectTimeout:4000000000 concurrentConns:1 sessCtlTimeout:0 postFunc:<nil> clientConfigOpts:<nil> verifyError:<nil>}
        --- FAIL: TestIntegrations/Disconnection/expired_cert_node_recording (7.05s)
```

--- a/integration/integration_test.go
+++ b/integration/integration_test.go
@@ -2130,6 +2130,7 @@ func testDisconnectScenarios(t *testing.T, suite *integrationTestSuite) {
 				MaxSessionTTL:         types.NewDuration(2 * time.Second),
 			},
 			disconnectTimeout: 4 * time.Second,
+			clientConfigOpts:  func(cc *helpers.ClientConfig) { cc.DisableSSHResumption = true },
 		},
 		{
 			name:          "expired cert proxy recording",

